corpus/filter.py

```diff
@@ -174,16 +174,18 @@ def run(self):
 
 
 class FilterCorpusBySegmentsJob(Job):
-    def __init__(self, corpus_file, segment_list, compressed=False, invert_match=False):
+    def __init__(
+        self, bliss_corpus, segment_file, compressed=False, invert_match=False
+    ):
         """
-        :param Path corpus_file:
-        :param Path segment_list:
+        :param Path bliss_corpus:
+        :param Path segment_file:
```

Comment:
This job was written with a list of segment files in mind and later extended to include the option of having a single segment file.

I agree that the "single file" case can be the expected default and we can name the parameter `segment_file`, but we should include the possibility to give a list of files in the doc string

Reply:
I would go for `segment_files` then instead.